Student/Student/Assignments/AssignmentDetails/AssignmentDetailsPresenter.swift

```diff
@@ -183,6 +183,7 @@ class AssignmentDetailsPresenter: PageViewLoggerPresenterProtocol {
             Submission.save(submission, in: context)
             try? context.save()
         }
+        refresh()
```

Comment:
The submission status was not updating after file upload submissions completed so refresh on that notification.